docs/changelogs/1.8.x/1.8.4.md

```diff
@@ -0,0 +1,174 @@
+* Fixed Iron Hands and Iron Bundle incorrect backsprite palettes @cafei-uh in https://github.com/rh-hideout/pokeemerald-expansion/pull/4552
+* Fixed transparent pixel in Mega Mewtwo Y's backsprite eye by @TheJamesLJ in https://github.com/rh-hideout/pokeemerald-expansion/pull/4541
+* Fixed floating Munkidori backsprite @cafei-uh in https://github.com/rh-hideout/pokeemerald-expansion/pull/4552
+* Fixed Salandit's sprite and palettes by @BlueAnthem37510 in https://github.com/rh-hideout/pokeemerald-expansion/pull/4614
+* Fixed regression from #4230 (1.8.0) that caused Mr. Mime's typing to ignore `P_UPDATED_TYPES` by @PhallenTree in https://github.com/rh-hideout/pokeemerald-expansion/pull/4547
+* Fixed silent Oricorio Pom-Pom cry by @cafei-uh in https://github.com/rh-hideout/pokeemerald-expansion/pull/4551
+* Fixed regression from #4360 (1.8.2) that caused Shaymin to changing form with the incorrect status + receiving damage upon form change by @TheJamesLJ in https://github.com/rh-hideout/pokeemerald-expansion/pull/4559
+
+## ⚔️ Battle General ⚔️ ##
+### Fixed
+* Fixed Imposter and Transform causing the user to look shiny until screen is refreshed if they're not in the first party slot by @Sneed69 in https://github.com/rh-hideout/pokeemerald-expansion/pull/4486
+* Fixed regression in #4307 (1.8.2) that caused special battle types to not finish correctly by @ghoulslash in https://github.com/rh-hideout/pokeemerald-expansion/pull/4536
+* Fixed multiple Speed ignoring effects by @Sneed69 in https://github.com/rh-hideout/pokeemerald-expansion/pull/4613
+    * Fixed Quick Draw always outspeeding Quick Claw and Custap Berry.
+    * Fixed Myceleum Might always outspeeding Stall.
+
+## 🤹 Moves 🤹
+### Changed
+* Ported Explosion/Mind Blown/Steel Beam implementation from CFRU (with tweaks) by @AlexOn1ine in https://github.com/rh-hideout/pokeemerald-expansion/pull/4516
+### Fixed
+* Fixes Thief and Covet used by an opponent not stealing items by @PhallenTree in https://github.com/rh-hideout/pokeemerald-expansion/pull/4506
+* Fixed Substitute, Fillet Away, Shed Tail and Belly Drum missing their `HITMARKER_PASSIVE_DAMAGE`, which weretriggering effects where they shouldn't by @hedara90 in https://github.com/rh-hideout/pokeemerald-expansion/pull/4533
+    * Cleanup by @AsparagusEduardo in https://github.com/rh-hideout/pokeemerald-expansion/pull/4629
+* Fixed Copycat issues by @PhallenTree in https://github.com/rh-hideout/pokeemerald-expansion/pull/4549
+    * Copycat can no longer call Z-Moves.
+    * 2-turn moves no longer play their animation twice when called by Copycat.
+* Fixed Substitute issues by @hedara90 in https://github.com/rh-hideout/pokeemerald-expansion/pull/4558
+    * The following secondary effects no longer occur if the target is behind a Substitute.
+        * Knock Off
+        * Salt Cure
+        * Smack Down/Thousand Arrows
+        * Wake-Up Slap/Smelling Salts/Sparkling Aria (including doubling in damage)
```

Comment:
```suggestion
        * Wake-Up Slap/Smelling Salts (including doubling in damage)
```
Sparkling Aria still happens if the target is behind a Substitute since it's a sound move.

Reply:
Damn, true. Will quickly push a commit to fix this.